packages/orchestrator-ui-components/src/components/WfoKeyValueTable/styles.ts

```diff
@@ -24,6 +24,8 @@ export const getStyles = (theme: EuiThemeComputed) => {
         rowNumber % 2 ? lightBackground : darkBackground;
 
     const keyColumnStyle = css({
+        display: 'flex',
+        alignItems: 'center',
```

Comment:
This causes trouble when you have multiple lines in a key-value table.
Before your adjustments:
![image](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/20791917/224a0009-31dc-426a-8a28-0c58fd1c306a)
After your adjustments:
![image](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/20791917/fcd006b9-fb6e-4456-8e69-de6b042e5d39)

What are you trying to fix here?

Reply:
I wanted this change so I can put the "Default Sending Level" and "Customers to Inform" inside the Service Tickets Detail Table: 

Before:
![Screenshot 2024-03-18 at 14 07 28](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/61829525/fcd222bd-ac51-44e4-98be-99e38809c714)


Now:
![Screenshot 2024-03-18 at 14 03 46](https://github.com/workfloworchestrator/orchestrator-ui-library/assets/61829525/af6762f2-7408-4c58-a50e-358abf87c3d5)